content/outcomes/cloud-native-development/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cloud Native Development
+weight: -1
+layout: single
+description: "Learn the basics of cloud native development with Spring. This developer pre-engagement guide helps you to get the most of an engagement following the Tanzu Labs methodology."
+team:
+ - VMware Tanzu Labs
+---
+ 
+## Why is this important?
+
+This learning path will provide you with foundational knowledge around modern, cloud-native software development with Spring and Cloud-Foundry. It has been conceived as pre-work to get the most of an application-related software development engagement with Tanzu Labs. 
+
+## Who is this learning path for?
+ 
+This learning path is mainly for software engineers and other members of the technical development team involved in a Tanzu Labs engagement.
+
+
+## What you will learn
+ 
+Each section of this guide provides some context to the skills or technologies in question, with links throughout to specific and targeted info you will need. Each section is followed by a list of suggested homework and optional additional resources.
+
+Going through the material of this learning path should not take more than 8 hours. However, the more time you put in, the more you will get out of the engagement.
+ 
+The following topics are covered:
+ 
+**General skills**
+* [Things to unlearn](/outcomes/cloud-native-development/general__unlearn/)
+* [Twelve-Factor Apps](/outcomes/cloud-native-development/general__twelve-factor-apps/)
+* [Development Tools](/outcomes/cloud-native-development/general__dev_env_tools/)
+* [Spring Ecosystem](/outcomes/cloud-native-development/general__spring_ecosystem/)
+* [Test Driven Development](/outcomes/cloud-native-development/general__tdd/)
```

Comment:
When the TDD outcome (#959) s delivered we should figure what we want to do with these two versions of similar content.

Reply:
Good point!

Updated by efecc6f1867f041d65823872775ea11e72a858dd